exercise5_code.R

```diff
@@ -0,0 +1,23 @@
+setwd("C:/Users/DAVIS/Desktop/shell-novice-data/exer5/")
+data=read.table(file="test.dat",header=FALSE, sep=" ")
+
+wages = read.csv(file="wages.csv", header=TRUE, stringsAsFactors = FALSE)
+
+#Challenge question 1
+test1 <- unique(wages[,c("gender","yearsExperience")])

```

Comment:
try to avoid names like 'test1' and 'test2.' It will make your code confusing when you have to come back to it someday.